js/background.js

```diff
@@ -85,3 +85,17 @@ chrome.omnibox.onInputEntered.addListener( function(text) {
         chrome.tabs.update(tabs[0].id, {url: "https://duckduckgo.com/?q="+encodeURIComponent(text)});
     });
 });
+
+//This adds Context Menu when user select some text.
+//create context menu
+chrome.contextMenus.create({
+    title : 'Search anonymously on DuckDuckGo',
```

Comment:
Yes, it is possible to add selection text in context menu. I am requested for new PR